frontend/src/assets/css/variables.scss

```diff
@@ -15,20 +15,18 @@
 // (Using a different license header format here because scss-to-json doesn't
 // like multi-line comments)
 
-// 1rem == 16px
-// (Can't set actual pixels here because it breaks reboot)
-$font-size-base: 1rem;
+$font-size-base: 18px;
```

Comment:
The problem is that there's no round pixel-based size for a small font that is both readable and nicely divisible by 18px:

16/18 = 0.888...
15/18 = 0.8333...
14/18 = 0.777...
13/18 = 0.7222...
12/18 = 0.666...
etc.

So this looks like a tradeoff between two types of accessibility: (1) allowing people to improve readability by increasing the base font size (rather than zooming), and (2) allowing people to have fonts that have reasonable & non-fractional sizes by default.

So I looked around at [the top websites in the US](https://www.semrush.com/blog/most-visited-websites/) and most choose to drop (1) in favor of (2).
* ✘ Google - sets fonts using px, so doesn't support (1)
* ✘ YouTube
* ✘ Facebook
* ✘ Amazon
* ✅ Wikipedia
* ✘ Yahoo
* ✘ Reddit

(The next one was PornHub so I stopped there :laughing:)

Given that most websites don't support this, my guess is very few people / almost no-one are actually relying on the browser's default font size for accessibility purposes. Rationale: they wouldn't be able to use most of the web if they did — so they must be relying on other technologies.

With all this in mind, I propose we can go ahead with this change and use px-based fonts.

Reply:
px-based fonts is fine (another option is to keep it 16px, and set `body` to `1.125rem`), but it's probably overkill now.